### PR TITLE
Fixes electric chairs, makes electric chairs more flashy

### LINKED ISSUE #6149: Electric chairs don't work
You can turn them on with the right clic verb, but nothing else
I never saw a working one anyways
#3107 related

### PR DESCRIPTION
**Pull request by 9600bauds**
There was a whole load of oldcode in these, I kind of explained what I did in the comments I guess

//I guess at some point, this shock kit thing was meant to be an /obj/item/DEVICE/assembly/ with it's own radio_frequency datum,
//but honestly I think just using the electropack for the whole signal-receiving thing works out much better.

Fixes #6149 

Whebbum: http://a.pomf.cat/zwdtnk.webm

### CODE REVIEW

**Comment by Zth--** [comment]
Yessss :+1:

File: code/game/objects/structures/electricchair.dm [hunk 1]
```diff
@@ -2,13 +2,26 @@
 	name = "electric chair"
 	desc = "Looks absolutely SHOCKING!"
 	icon_state = "echair0"
-	var/on = 0
+	var/on = 1
 	var/obj/item/assembly/shock_kit/part = null
 	var/last_time = 1.0
+	var/datum/effect/effect/system/spark_spread/spark_system
+
+/obj/item/assembly/shock_kit/New()
```
Comment (9600bauds) [inline code/game/objects/structures/electricchair.dm]: wait a sec

**Comment by ghost** [comment]
:+1:

**Comment by Imagebard** [comment]
Does this fix using electropacks in your back slot?

**Comment by 9600bauds** [comment]
???
I'm not sure what that is, there's no bug report on it

**Comment by Imagebard** [comment]
I'm going to assume it doesn't, then.

But if you're interested in fixing it:
http://a.pomf.se/olepyx.webm

**Comment by 9600bauds** [comment]
Link won't load for some reason, what is it?

**Comment by Imagebard** [comment]
You're supposed to be able to put Electropacks on someone's else back
They're supposed to need help form someone else to take it out, and you should be able to shock them as if they were in a electric chair.

But its bugged:
I'm pretty sure it doesn't work at all
You can try to place a backpack in your back slot and it'll replace the electropack which you aren't supposed to take out yourself

**Comment by 9600bauds** [comment]
Inventory code is wonky as hell and this is getting really annoying to try to fix
Probably make a bugreport and revisit it some other time or maybe have someone else try to fix it

**Pull request merged**